flow/metrics.yaml

```diff
@@ -30,6 +30,17 @@ components:
             Latency metrics.
           $ref: '#/components/schemas/Flow.Latency.Metrics'
           x-field-uid: 4
+        egress:
+          description: |-
+            Enumerate flow metrics by describing the shape of the packet as it
+            exits the system under test and assigning unique names to
+            individual packet header fields.
+          type: array
+          items:
+            $ref: './egress/egress.yaml#/components/schemas/Egress.Header'
+          x-field-uid: 8
```

Comment:
What's the reason for value 8 and not 5?   Were the values 5, 6 & 7 used earlier?